src/modules/core/components/CoreSwiper/index.tsx

```diff
@@ -31,7 +31,6 @@ const CoreSwiper: FC<CoreSwiperProps> = ({
     effect="fade"
     fadeEffect={{ crossFade: true }}
     rewind
-    loop
```

Comment:
In case of using "loop" the following error occurs while resizing the window:
![image](https://github.com/Vadim-Kostetskyi/Online-store/assets/95620370/59454266-faf5-48df-8e9a-44b63037dcc3)

It is mentioned here:
"Uncaught error when unmounting Swiper with loop on window resize #7265"
https://github.com/nolimits4web/swiper/issues/7265 

Also it seems the functionality has not been impacted by removing the "loop".